### DISCUSSION TITLE
Why does minio recommends 4 nodes with 2 disks per node for production setup?

### DISCUSSION THREAD

**Discussion by weiklr** [minio/minio#12006] [Q&A]
hi, with reference to the sizing guide here: https://github.com/minio/minio/blob/master/docs/distributed/SIZING.md ,

can anyone explain why it's recommended to have at least 4 node with 2 drives per node for a distributed setup, and not a 4 node with 1 drive per node?

**Comment by harshavardhana** [comment]
Because all other setups are toy setups with not enough redundancy
.

**Reply by weiklr** [reply]
can I clarify what you mean by not enough redundancy, as both have the same number of tolerance for reads and writes based on the sizing guide?

**Reply by harshavardhana** [reply]
Yes but capacity wise for us to support it doesn't make sense in production deployments, if you are planning to manage it on your own you can deploy with even 2 nodes.
